Settings.cs

```diff
@@ -52,6 +54,18 @@ public static int frameRate
 			get => data.frameRate;
 			set => data.frameRate = value;
 		}
+		
+		public static double hostTransparency
+		{
+			get => data.hostTransparency;
+			set => data.hostTransparency = value;
+		}
+		
+		public static bool hostTransparent
+		{
+			get => data.hostTransparent;
+			set => data.hostTransparent = value;
+		}
 

```

Comment:
This value can be calculated and does not need to be saved. NIT: Should be prefixed with "is" to avoid confusion with the `hostTransparency` property

```C#
public static bool hostTransparent => data.hostTransparency != 1;
```

Reply:
using `isHostTransparent` and `hostOpacity` which are more clear